### DISCUSSION TITLE
Bouncy hair

### DISCUSSION THREAD

**Discussion by inTIMidation297** [keiwando/toonsquid#643] [Show and tell]
https://github.com/user-attachments/assets/39ea5f11-dfd1-40f1-8f57-e1dd6ee19074

**Comment by budaloops** [comment]
Looking good! I’d like to see where it goes from here

**Comment by inTIMidation297** [comment]
Ty! I'm still learning toonsquid. I love the app so much and it gave me
motivation to fulfil my dream of being an animator one day
